pkg/daemon/ceph/client/pool.go

```diff
@@ -190,6 +191,13 @@ func CreatePoolForApp(context *clusterd.Context, clusterName string, newPool Cep
 		return fmt.Errorf("failed to create pool %s. %+v", newPool.Name, err)
 	}
 
+	// enable EC overwrites for an EC pool
+	if isECPool {
+		if err = SetPoolProperty(context, clusterName, newPool.Name, "allow_ec_overwrites", "true"); err != nil {
```

Comment:
Instead of setting this for all EC pools, what if the `CreatePoolForApp` tool a bool arg that indicates whether to allow ec overwrites? The default would pass `true` and object storage would call `false`. Then the object store wouldn't need to change the value again after initial creation.